CreateMessage/handler.ts

```diff
@@ -286,6 +307,14 @@ export function CreateMessageHandler(
     const { service, email: serviceUserEmail } = userAttributes;
     const { authorizedRecipients, serviceId } = service;
 
+    // in order to send attachments with a message the message should be ADVANCED
+    if (
+      messagePayload.feature_level_type === FeatureLevelTypeEnum.STANDARD &&
```

Comment:
I'd give a name to complex conditionals in order to make the code more redable ie.

`const canSendAttachments = messagePayload.feature_level_type === FeatureLevelTypeEnum.STANDARD && messagePayload.content.third_party_data.has_attachments === true;
`

Reply:
moreover this would make the comment superfluos. 

another consideration: according to the comment above, the condition should be instead

onst canSendAttachments = messagePayload.feature_level_type !== FeatureLevelTypeEnum.ADVANCED && messagePayload.content.third_party_data.has_attachments === true;

Reply:
very good!